package.json

```diff
@@ -40,6 +40,9 @@
     "@heroicons/react": "~2.1.1",
     "@mdx-js/mdx": "^3.0.0",
     "@nodevu/core": "~0.1.0",
+    "@orama/highlight": "^0.1.2",
+    "@orama/orama": "^2.0.0",
+    "@oramacloud/client": "^1.0.2",
```

Comment:
Which of these dependencies are only executed on the build script?

Reply:
Probably just `@orama/orama`, for types. I can move it to `devDependencies`